### PR TITLE
fix Sass lint errors

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
Fixes warnings from the `yarn lint:sass` command

## Testing done
Local

## Screenshots
image

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/sass/vaos.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
 .vaos-landing__icon {
   display: inline-block;
   font-size: 19px;
-  color: white;
```
Comment (erikphansen) [inline src/applications/vaos/sass/vaos.scss]: Had to change this from `white` to `#fff` to quiet the `no-color-keywords` error. But we still get a `no-color-literals` error. So, what do we do? Make a variable named `$white` to make all the rules happy? Seems pretty silly.

Comment (rianfowler) [reply src/applications/vaos/sass/vaos.scss]: maybe $color-white: https://design.va.gov/design/color-palette

Comment (erikphansen) [reply src/applications/vaos/sass/vaos.scss]: haha, yeah, I noticed that after your Slack comment. 🤦‍♂

File: src/applications/edu-benefits/sass/edu-benefits.scss [hunk 2]
```diff
@@ -116,7 +116,7 @@
       }
 
       .institution-information {
-        background-color: #f1f1f1;
+        background-color: $color-gray-lightest;
```
Comment (erikphansen) [inline src/applications/edu-benefits/sass/edu-benefits.scss]: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L71

File: src/applications/facility-locator/sass/facility-locator.scss [hunk 3]
```diff
@@ -129,7 +135,7 @@
       text-decoration: none;
 
       &:focus {
-        outline: 2px solid #f9c642;
+        outline: 2px solid $color-gold-light;
```
Comment (erikphansen) [inline src/applications/facility-locator/sass/facility-locator.scss]: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L85

File: src/applications/facility-locator/sass/facility-locator.scss [hunk 4]
```diff
@@ -677,17 +683,17 @@
 
 
 #infoBubble {
-  background-color: #e1f3f8;
+  background-color: $color-aqua-lightest;
```
Comment (erikphansen) [inline src/applications/facility-locator/sass/facility-locator.scss]: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L41

File: src/applications/facility-locator/sass/facility-locator.scss [hunk 5]
```diff
@@ -677,17 +683,17 @@
...
   color: inherit;
   text-align: left;
   width: 115%;
   position: absolute;
   top: -7.1rem;
   right: -7.1rem;
   padding: 0.7rem;
-  box-shadow: 0px 0px 4px #808080;
+  box-shadow: 0px 0px 4px $color-gray-medium;
```
Comment (erikphansen) [inline src/applications/facility-locator/sass/facility-locator.scss]: close enough https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L72

File: src/applications/personalization/dashboard/sass/messaging/partials/_message-compose.scss [hunk 6]
```diff
@@ -1,3 +1,5 @@
+$color-close-button: rgba($color-base, .8);
```
Comment (erikphansen) [inline src/applications/personalization/dashboard/sass/messaging/partials/_message-compose.scss]: I'm pretty sure that the `.msg-attachment-close` class that uses this color can be completely removed from the codebase. It's hard to know with 100% certainty, but I'm not seeing any usage of that class anywhere.

File: src/applications/personalization/profile360/sass/profile-360.scss [hunk 7]
```diff
@@ -39,7 +39,7 @@ input.usa-only-phone {
   line-height: 1;
   color: $color-black;
   text-align: center;
-  background-color: #eee;
+  background-color: $color-gray-lightest;
```
Comment (erikphansen) [inline src/applications/personalization/profile360/sass/profile-360.scss]: close enough https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L71

File: src/platform/forms/sass/_m-schemaform.scss [hunk 8]
```diff
@@ -642,7 +642,7 @@ button:focus,
 input:focus,
 select:focus,
 textarea:focus {
-  outline: 2px solid #f9c642;
+  outline: 2px solid $color-gold-light;
```
Comment (erikphansen) [inline src/platform/forms/sass/_m-schemaform.scss]: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L85

File: src/platform/site-wide/sass/modules/_m-crisis-line.scss [hunk 9]
```diff
@@ -1,8 +1,3 @@
-// These should be replaced when the new colors are in the design
```
Comment (erikphansen) [inline src/platform/site-wide/sass/modules/_m-crisis-line.scss]: these color vars do exist in the design system https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L131-L132

File: src/platform/site-wide/sass/modules/_m-crisis-line.scss [hunk 10]
```diff
@@ -103,7 +98,7 @@ button.va-crisis-line {
     max-height: 100%;
   }
   &-title {
-    color: #212121;
+    color: $color-base;
```
Comment (erikphansen) [inline src/platform/site-wide/sass/modules/_m-crisis-line.scss]: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L51

File: src/platform/site-wide/sass/modules/_m-cta-widget.scss [hunk 11]
```diff
@@ -4,7 +4,7 @@
     border-left: none;
 
     .fa-angle-down {
-      color: #323A45;
+      color: $color-gray-dark;
```
Comment (erikphansen) [inline src/platform/site-wide/sass/modules/_m-cta-widget.scss]: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L73

**Review by ncksllvn — APPROVED** [review]
Thank you for doing this!

File: src/applications/facility-locator/sass/facility-locator.scss [hunk 12]
```diff
@@ -4,6 +4,12 @@
 @import "~leaflet/dist/leaflet.css";
 @import "~react-tabs/style/react-tabs";
 
+// TODO: These colors are not close enough to anything that already exists in
+// Formation. Perhaps the design should be reevaluated to see if Formation
+// colors could be used in place of these
```
Comment (cvalarida) [inline src/applications/facility-locator/sass/facility-locator.scss]: Probably worth writing a ticket for this so we can track it.

File: src/platform/site-wide/sass/modules/_m-homepage.scss [hunk 13]
```diff
@@ -212,7 +212,7 @@
     &.fa-user-circle {
       font-size: 1.5em;
       padding-right: .3em;
-      
+
```
Comment (cvalarida) [inline src/platform/site-wide/sass/modules/_m-homepage.scss]:
```suggestion
```

**Review by cvalarida — APPROVED** [review]

File: src/platform/site-wide/sass/modules/_m-homepage.scss (hunk 13 above)
Comment (erikphansen) [reply src/platform/site-wide/sass/modules/_m-homepage.scss]: good catch. I didn't even notice this as my editor is the one who removed the trailing whitespace.

File: src/applications/facility-locator/sass/facility-locator.scss (hunk 12 above)
Comment (erikphansen) [reply src/applications/facility-locator/sass/facility-locator.scss]: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/1689

**Pull request merged**